page/code-organization/deferreds/examples.md

```diff
@@ -158,7 +158,7 @@ The new `$.afterDOMReady()` helper method provides proper timing after the DOM i
 
 While jQuery offers all the event binding one may need, it can become a bit cumbersome to handle events that are only supposed to be dealt with once.
 
-For instance, you may wish to have a button that will open a panel the first time it is clicked and leave it open afterwards or take special initialization actions the first time said button is clicked. When dealing with such a situation, one usually end up with code like this:
+For instance, you may wish to have a button that will open a panel the first time it is clicked and leave it open afterward or take special initialization actions the first time said button is clicked. When dealing with such a situation, one usually ends up with code like this:
```

Comment:
The word _afterwards_ is a British English spelling.  In the interest of consistency, this replaces it with the American English spelling.
https://github.com/jquery/api.jquery.com#spelling

The plural verb _end_ does not agree with the singular subject _one_.  This changes the verb form for subject-verb agreement.